cpp/cppsdk/gsdk.cpp

```diff
@@ -155,6 +155,11 @@ namespace Microsoft
                     logFolder = "";
                 }
 
+                if (logFolder.back() != '/' && logFolder.back() != '\\')
+                {
+                    logFolder.append("/");
```

Comment:
```suggestion
                    logFolder.append(System.IO.Path.DirectorySeparatorChar);
```

Reply:
System.IO.Path.DirectorySeparatorChar is part of .Net Framework, which is not useable in C++